packages/styled-components/src/test/constants.test.js

```diff
@@ -12,6 +9,8 @@ describe('constants', () => {
 
   describe('SC_ATTR', () => {
     function renderAndExpect(expectedAttr) {
+      const React = require('react');
+      const TestRenderer = require('react-test-renderer');
```

Comment:
The `jest.resetModules()` call would break things without re-requiring the packages as the `react` would still hold a reference to the dispatcher that was imported at the top level.